airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcCtidInitializer.java

```diff
@@ -172,6 +172,10 @@ public static List<AutoCloseableIterator<AirbyteMessage>> cdcCtidIteratorsCombin
 
       // Gets the target position.
       final var targetPosition = PostgresCdcTargetPosition.targetPosition(database);
+      // Attempt to advance LSN past the target position. For versions of Postgres before PG15, this
+      // ensures that there is an event that debezium will
+      // receive that is after the target LSN.
+      PostgresUtils.advanceLsn(database);
```

Comment:
should we make that only execute on postgres <15 ? That allows us to request less access right post 15, no?

Reply:
If we don't have the correct access, we'll just fail in the `advanceLsn` method and log that scenario. Success/failure of this advancement should not block the sync from proceeding. 

For PG14, the query that I've added in this PR requires no additional permissions. For PG15+, this same query will just fail because of permissions issues and move on. 

Phase 2 hopes to address this with additional steps for the user to setup a heartbeat table

Reply:
Change will make things better for 14 and below.
But for 15 and above even if we can make this query, it will not be effective in bumping the LSN.
So maybe it makes sense to avoid a constant error or questions from customers about it.
We already have code doing "legacy" ctid for postgres <14.
You can utilize that function to do things <15

Reply:
See `CtidUtils.isTidRangeScanCapableDBServer()`

Reply:
Added a check